### DISCUSSION TITLE
Working thrru corporate proxy - a hack to make it work but it didn't make me feel good.

### DISCUSSION THREAD

**Discussion by MontyTHall** [CopilotC-Nvim/CopilotChat.nvim#77] [General]
I was getting this:

requests.exceptions.SSLError: HTTPSConnectionPool(host='api.githubcopilot.com', port=443): Max retries exceeded with url: /chat/completions (Caused by SSLError(CertificateError("hostname 'api.githubcopilot.com' doesn't match either of '*.gist.githubusercontent.com', 'gist.githubusercontent.com
'
After messing around I did this (copilot.py:127):
        response = self.session.post(
            url, headers=self._headers(), json=data, stream=True, verify=False
        )
I had to add verify=False.  Is there a proper way to handle this?

**Comment by jellydn** [comment]
Answered by @gptlang on https://discord.com/channels/1200633211236122665/1209114762584002570/1211707411824377896
